java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/build/LookupBuildInfoCommand.java

```diff
@@ -302,6 +328,64 @@ private void doBuildAnalysis(String scmUrl, String scmTag, String context, Build
                 info.setAllowedDifferences(buildRecipeInfo.getAllowedDifferences());
                 Log.infof("Got build recipe info %s", buildRecipeInfo);
             }
+            if (sharedRegistries != null) {
```

Comment:
I would also check the main rebuilt repo here, so it is not treated differently to the shared ones. 

This would also be useful for testing, you could run the existing tests, then in a new namespace reference the same repo and make sure it picks up the artifacts from the first test.